lib/ansible/parsing/vault/__init__.py

```diff
@@ -68,13 +68,8 @@
     from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
     from cryptography.hazmat.backends import default_backend
     HAS_PBKDF2HMAC = True
-except ImportError:
+except:
```

Comment:
I was considering adding a case for VersionConflict. When would you want an optional import to raise an exception? Should a warning be printed?